libs/build_dod_dataset.py

```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
 
 
 def _get_usa_by_county_df():
-    # TODO: read this from a dataset class
-    url = "{}/data/cases-jhu/csse_covid_19_daily_reports/{}.csv".format(
-        get_public_data_base_url(), latest.strftime("%m-%d-%Y")
-    )
-    raw_df = pd.read_csv(url, dtype={"FIPS": str})
-    raw_df["FIPS"] = raw_df["FIPS"].astype(str).str.zfill(5)
-
-    column_mapping = {
-        "Province_State": "Province/State",
-        "Country_Region": "Country/Region",
-        "Last_Update": "Last Update",
-        "Lat": "Latitude",
-        "Long_": "Longitude",
-        "Combined_Key": "Combined Key",
-        "Admin2": "County",
-        "FIPS": "State/County FIPS Code",
-    }
-    remapped_df = raw_df.rename(columns=column_mapping)
-
-    # USA only
-    us_df = remapped_df[(remapped_df["Country/Region"] == "US")]
-    jhu_column_names = [
-        "Province/State",
-        "Country/Region",
-        "Last Update",
-        "Latitude",
-        "Longitude",
-        "Confirmed",
-        "Recovered",
-        "Deaths",
-        "Active",
-        "County",
-        "State/County FIPS Code",
-        "Combined Key",
-        # Incident rate and people tested do not seem to be available yet
-        # "Incident Rate",
-        # "People Tested",
-    ]
-    final_df = pd.DataFrame(us_df, columns=jhu_column_names)
-    final_df["Last Update"] = pd.to_datetime(final_df["Last Update"])
-    final_df["Last Update"] = final_df["Last Update"].dt.strftime("%-m/%-d/%Y %H:%M")
-
-    final_df["County"] = final_df["County"].replace(county_replace_with_null)
-    final_df["Combined Key"] = final_df["Combined Key"].str.replace("Unassigned, ", "")
-    final_df = final_df.fillna(NULL_VALUE)
-    final_df = final_df.drop_duplicates(
-        "State/County FIPS Code"
-    )  # note this is a hack, 49053 is dupped in JHU data :(
-    final_df.index.name = "OBJECTID"
-    # assert unique key test
-    assert final_df["Combined Key"].value_counts().max() == 1
-    assert final_df["State/County FIPS Code"].value_counts().max() == 1
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.COUNTY)
+
 
-    return final_df
+def _get_usa_by_states_df():
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.STATE)
 
 
 def get_usa_by_county_with_projection_df(input_dir, intervention_type):
     us_only = _get_usa_by_county_df()
-    fips_df = FIPSPopulation.local().data  # used to get interventions
-    interventions_df = (
-        _get_interventions_df()
-    )  # used to say what state has what interventions
+    fips_df = FIPSPopulation.local().data
```

Comment:
do we need to specify this is county data?

Reply:
it works okay because the left join is only starting with counties, but might be good to make it explicit.